### ISSUE TITLE
show_transectmatrix needs a neutral color scheme

### ISSUE THREAD

**Issue by fawda123** [tbep-tech/tbeptools#38]
Breakpoints for r/o/y/g probably don't have any ecological significance.

**Comment by fawda123** [comment]
@Gmangrove thoughts? neutral enough? neutral too much??

![image](https://user-images.githubusercontent.com/3086881/108269601-38cf0580-713c-11eb-8943-3b9477bb187e.png)

**Comment by Gmangrove** [comment]
I don't think there's enough difference between the tiers, almost like 1-2 blend, same with 2-3, 3-4. Maybe something along these lines (but still trying to avoid the happy & danger colors): https://docs.google.com/presentation/d/1Ss3ddm2qMYPPKO5qh6qkL6IQRAEmLEk0Pu-oRVdA1Fc/edit?usp=sharing

**Comment by fawda123** [comment]
@Gmangrove I think it might be better to avoid the tiers all together since the groups are based on arbitrary breakpoints.  That's why I went with a continuous color ramp with only one color.  I think the most we should try to do with this graph is indicate relative changes and the single color ramp could work well for this.  

I made the bottom end a bit more noticeable, does that help? 
![image](https://user-images.githubusercontent.com/3086881/108406896-016e6080-71f1-11eb-8186-5404ad2e18e4.png)
